src/app/modules/main/profile_section/advanced/controller.nim

```diff
@@ -76,6 +76,25 @@ method toggleCommunityHistoryArchiveSupport*(self: Controller) =
 method isCommunityHistoryArchiveSupportEnabled*(self: Controller): bool =
   return self.nodeConfigurationService.isCommunityHistoryArchiveSupportEnabled()
 
+method toggleWakuV2Store*(self: Controller) =
+  let enabled = self.nodeConfigurationService.isWakuV2StoreEnabled()
+
+  if enabled:
```

Comment:
@richard-ramos can we just rewrite this `if` (both `then` and `else` blocks) in a bit cleaner way, like:
```
method toggleWakuV2Store*(self: Controller) =
  let enabled = self.nodeConfigurationService.isWakuV2StoreEnabled()
  if (not self.nodeConfigurationService.setWakuV2StoreEnabled(not enabled)):
    # in the future we may do a call from here to show a popup about this error
    error "an error occurred, we couldn't enable wakuv2 store"
    return
  self.delegate.onWakuV2StoreToggled()
```